packages/drivers/routerlicious-driver/src/errorUtils.ts

```diff
@@ -31,13 +32,26 @@ export const RouterliciousErrorTypes = {
 	 * SSL Certificate Error.
 	 */
 	sslCertError: "sslCertError",
+
+	/**
+	 * Error for when the service drains a cluster to which the socket connection is connected to and it disconnects
+	 * all the clients in that cluster.
+	 */
+	clusterDrainingError: "clusterDrainingError",
 } as const;
 /**
  * @internal
  */
 export type RouterliciousErrorTypes =
 	(typeof RouterliciousErrorTypes)[keyof typeof RouterliciousErrorTypes];
 
+/**
+ * Error code for when the service drains a cluster to which the socket connection is connected to and it disconnects
+ * all the clients in that cluster.
+ * @internal
+ */
+export const R11sServiceClusterDrainingErrorCode = "ClusterDraining";
```

Comment:
I'd suggest moving all the pieces that are part of contract (i.e. changing them will break protocol) into contract.ts or similar file, so that implications of changing such things is very obvious to reader.

Reply:
done